### PR TITLE
Feature/newtheme

### PR DESCRIPTION
**Pull request by banhae**
added new theme

### CODE REVIEW

File: Main.cs [hunk 1]
```diff
@@ -139,32 +120,50 @@ private void GenerateThumb()
             }
 
             // 1) some setup
-            var tmpImage = (Image)cachedImage.Clone();
+            var tmpImage = new Bitmap(cachedImage.Width, cachedImage.Height);
+            var copyImage = (Image)cachedImage.Clone();
+
+            var backgroundColor = new SolidBrush(Color.FromArgb(42, 42, 42));
```
Comment (popekim) [inline Main.cs]: i think both newsolidbrush and pen are IDisposable..

if so please use

using (var backgroundColor = new SolidBrush(Color.FromArgb(42, 42, 42))
using (var backgroundEdgeColor = new Pen(Color.FromArgb(42, 42, 42)))
{
}

File: Main.cs [hunk 2]
```diff
@@ -139,32 +120,50 @@ private void GenerateThumb()
...
+            var tmpImage = new Bitmap(cachedImage.Width, cachedImage.Height);
+            var copyImage = (Image)cachedImage.Clone();
+
+            var backgroundColor = new SolidBrush(Color.FromArgb(42, 42, 42));
+            var backgroundEdgeColor = new Pen(Color.FromArgb(42, 42, 42));
+
+            int thumbWidth = tmpImage.Width;
+            int thumbHeight = tmpImage.Height;
+
+            const int margin = 75;
```
Comment (popekim) [inline Main.cs:129]: margin should be % of either width or height?

File: Main.cs [hunk 3]
```diff
@@ -139,32 +120,50 @@ private void GenerateThumb()
...
-            // 2) calculate proper draw region based on ratio
-            // some youtube thumbs are shown in 4:3 ratio. so let's make it 4:3 ratio
-            float marginPercentW = 0;
-            if (tmpImage.Height * 16 == tmpImage.Width * 9)
-            {
-                marginPercentW = 4 * 0.5F / 16F;
-            }
+            var rect = new RectangleF(margin,
+                margin * 3,
+                thumbWidth/2 - margin * 3,
```
Comment (popekim) [inline Main.cs]: ctrl k +d

**Review by popekim — CHANGES_REQUESTED** [review]

File: Main.cs (hunk 1 above)
Comment (popekim) [reply Main.cs]: do the same for any IDisposable object please

File: Main.cs (hunk 2 above)
Comment (banhae) [reply Main.cs:129]: It's no problem. because, always get every thumbnails height 1,080 size.

**Pull request merged**